build.sbt

```diff
@@ -1,17 +1,58 @@
+import sbt.Keys._
+
 name := "live-stream-creator"
 
 version := "1.0"
 
+/*i am not sure if we need this or not?*/
+//lazy val root = (project in file(".")).enablePlugins(PlayScala)
```

Comment:
`lazy val root` is just a little bit nicer as there are no backticks. Lets choose one or the other and delete the unused one.